liam2/exprtools.py

```diff
@@ -316,8 +317,7 @@ def _parse(s, interactive=False):
     if not isinstance(s, basestring):
         return s
 
-    # this prevents any function named something ending in "if"
-    str_to_parse = s.replace('if(', 'where(')
+    str_to_parse = re.sub(r'\b(if\()', 'where(', s)
```

Comment:
Given the exact definition of \b that I just looked at, I think it is correct, but since I never actually used \b, I will probably not remember it, so adding a (short) comment to what it does and/or a test with an arithmetic operator before the if, would appease my fear of breaking constructs like "whatever+if(cond, value2, value3)" which I have seen in a few places.